src/styles/_questionnaire.scss

```diff
@@ -133,6 +140,19 @@
       @extend .input;
       background-color: $secondaryColor;
       color: $fontColor;
+      &:hover {
+        background-color: $secondaryColor;
+        transform: none;
+        transition: none;
```

Comment:
Whai?

Reply:
When disabled we dont want inputs to animate, this gives user understanding that you cannot change it without clicking edit

Reply:
KUL